quickstart/azure/setup.md

```diff
@@ -72,6 +73,7 @@ To use a Service Principal, you must first create one.  This can be done using t
 Please refer to the Azure documentation for detailed instructions:
 
 * [Using the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli?view=azure-cli-latest)
+* Alternatively you can use [Azure Cloud Shell](https://shell.azure.com/) if you do not want to install Azure CLI 
```

Comment:
It'd also be nice to update the sentence above, e.g.:

```diff
- To use a Service Principal, you must first create one.  This can be done using the Azure CLI or the Azure Portal.
+ To use a Service Principal, you must first create one.  This can be done using the Azure CLI, the Azure Cloud Shell, or the Azure Portal.
```